src/rating/Rating.js

```diff
@@ -69,28 +69,40 @@ export default class Rating extends Component {
         this.setState({ value: gesture.dx });
       },
       onPanResponderRelease: event => {
-        onFinishRating(this.getCurrentRating());
+        const rating = this.getCurrentRating();
+        if (!fractions) {
+          // "round up" to the nearest star/rocket/whatever
+          this.setCurrentRating(rating);
```

Comment:
> I'm probably nitpicking here, but this small discrepancy may still confuse some users.

I love it. It's the minor details and polish that matters the most. 💯 

Also your gif editing skills are 🥇. Would be awesome to have these on the docs as well to show how fractions works.